cfg_template.yml

```
@@ -1,3 +1,7 @@
+prs:
+  # Change this if you use a different name for your default branch (e.g. main)
+  main-branch-name: master
```

Yeah, that sounds like a good idea, how about:

```suggestion
  # Uncomment this if you use a different name for your default branch (e.g. main)
  # main-branch-name: main
```